nucypher/config/constants.py

```diff
@@ -36,8 +36,8 @@
 
 # User Application Filepaths
 APP_DIR = AppDirs(nucypher.__title__, nucypher.__author__)
-DEFAULT_CONFIG_ROOT = APP_DIR.user_data_dir
-USER_LOG_DIR = APP_DIR.user_log_dir
+DEFAULT_CONFIG_ROOT = os.getenv('NUCYPHER_CONFIG_ROOT') or APP_DIR.user_data_dir
```

Comment:
**Optional:** you could simplify this to the following:
```suggestion
DEFAULT_CONFIG_ROOT = os.getenv('NUCYPHER_CONFIG_ROOT', default=APP_DIR.user_data_dir)
```

Same for the line below.